daml-lf/interpreter/src/main/scala/com/digitalasset/daml/lf/speedy/ValidateCompilation.scala

```diff
@@ -0,0 +1,119 @@
+// Copyright (c) 2021 Digital Asset (Switzerland) GmbH and/or its affiliates. All rights reserved.
+// SPDX-License-Identifier: Apache-2.0
+
+package com.daml.lf.speedy
+
+/**  ValidateCompilation (Phase of the speedy compiler pipeline)
+  *
+  *  This phases validates the final compilation result -- SExpr
+  */
+
+import com.daml.lf.speedy.SValue._
+import com.daml.lf.speedy.{SExpr => t}
```

Comment:
same remark here.

Reply:
Here I will avoid the prefix entirely, since validate will only operate on a single kind of expression: `SExpr`.